core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -135,21 +136,22 @@ var ExplorationEditorPage = function() {
         '.protractor-test-exploration-title-input'))
     );
 
-    await expTitle.sendKeys(title);
-    await expObjective.sendKeys(objective);
-
-    await element(by.css('.select2-container')).click();
-    await element(by.css('.select2-dropdown')).element(
-      by.css('.select2-search input')).sendKeys(category + '\n');
+    await action.sendKeys('Exploration Title', expTitle, title);
+    await action.sendKeys('Exploration Objective', expObjective, objective);
 
-    await element(by.css('.protractor-test-exploration-language-select'))
-      .click();
-    await element(by.css('.protractor-test-exploration-language-select'))
-      .sendKeys(language + '\n');
+    var container = element(by.css('.select2-container'));
+    await action.click('Container', container);
+    await action.sendKeys(
+      'Input search', element(by.css('.select2-dropdown')).element(by.css(
+        '.select2-search input')), category + '\n');
 
+    await action.click(
+      'Exp Language Select', selectExplorationLanguage);
```

Comment:
Unnecessary since `action.sendKeys` already clicks:

https://github.com/oppia/oppia/blob/5cdfd60ef5ccf150385c93d92158b89f3d5142a2/core/tests/protractor_utils/action.js#L53